examples/b2c/checkout/integrations/classes/B2BDeliverySample.cls

```diff
@@ -12,6 +12,9 @@ global class B2BDeliverySample implements sfdc_checkout.CartShippingCharges {
             // We're getting information like rates and carriers from this external service. 
             String siteLanguage = jobInfo.siteLanguage; 
 
+            // Following line of code calls the live HTTP Heroku Service to retrieve Shipping Methods.
+            // If you need to bypass the live call, use getShippingOptionsAndRatesFromMockedService instead.
+            // These implementations are just a sample with hardcoded Shipping options and MUST not be used in production systems.
```

Comment:
// Both implementations are just samples returning hardcoded Shipping options and MUST not be used in production systems.

Reply:
@gurpreetsainisalesforce It is better to add a note saying about what @wei-liu-salesforce mentioned about in our help document. We should use customers away it.